src/ghdcbot/engine/orchestrator.py

```
@@ -0,0 +1,535 @@
+            contribution_summaries_for_snapshot = None
+            list_summaries = getattr(self.storage, "list_contribution_summaries", None)
+            if callable(list_summaries):
+                try:
+                    contribution_summaries_for_snapshot = list_summaries(
+                        period_start,
+                        period_end,
+                        self.config.scoring.weights,
+                    )
+                except Exception:
+                    # If summaries can't be computed, snapshot will have empty contributors data
+                    pass
```

Ruff (0.15.5)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)